x-pack/test/saved_object_api_integration/common/lib/types.ts

```diff
@@ -21,11 +21,13 @@ export interface TestSuite<T> {
 export interface TestCase {
   type: string;
   id: string;
+  namespaces?: string[];
```

Comment:
As I mentioned above about defining namespaces in the `find` suite instead, if you made that change then this could be removed from the common interface as well.